site/en/docs/android/trusted-web-activity/quick-start/index.md

```diff
@@ -114,129 +112,20 @@ use PWA Builder to create an Android App that opens your PWA in
 
 ### A note on signing keys {: #a-note-on-signing-keys }
 
-Digital Asset Links take into account the key that an APK has been signed with and a common
-cause for verification failing is to use the wrong signature.
-(Remember, failing verification means you'll launch your website as a Custom Tab with
-browser UI at the top of the page.)
-When Bubblewrap builds the application, an APK will be created with a key setup during the `init`
-step.
+Digital Asset Links take into account the key that an APK has been signed with and a common cause for verification failing is to use the wrong signature. (Remember, failing verification means you'll launch your website as a Custom Tab with browser UI at the top of the page.) When Bubblewrap builds the application, an APK will be created with a key setup during the `init` step. However, when you publish your app in Google Play, another key may be created for you, depending on how you choose to handle signing keys. Learn more on [signing keys and how they relate to Bubblewrap and Google Play](/docs/android/trusted-web-activity/android-for-web-devs#upload-vs-signing-key).
 
 ## Setting up your asset link file {: #creating-your-asset-link-file }
 
+Digital Asset Links consist essentially of a file on your website that points to your app and some
+metadata in your app that points to your website.
+
 {% YouTube
   id='3bAQPnxLd4c'
 %}
 
-Besides the APK file, Bubblewrap output an `assetlinks.json` file that matches the key used to sign
```

Comment:
I approved another change which removed this section, so you'll probably need to rebase.

Reply:
Thanks for the heads up, Sam! I think I've resolved any conflicts with main now.